docs/getting-started/dockstore-tools.rst

```diff
@@ -13,46 +13,39 @@ Tutorial Goals
 
-If you followed :doc:`Getting Started With CWL </getting-started/getting-started-with-cwl>` to create a CWL tool, you will now have your ``Dockerfile`` and ``Dockstore.cwl`` in GitHub, have setup Quay.io to automatically build your Docker image, and have linked your accounts to Dockstore. This will allow you to use any of our registration methods, including legacy methods. Of course, you can always follow along with your own unique tools too, which may or may not be associated with an autobuilding Docker image.
+If you followed :doc:`Getting Started With CWL </getting-started/getting-started-with-cwl>` to create a CWL tool, you will now have your ``Dockerfile`` and ``Dockstore.cwl`` in GitHub, have setup Quay.io to automatically build your Docker image, and have linked your accounts to Dockstore. This will allow you to use any of our registration methods, including legacy methods. You can also follow this tutorial if your tool is not associated with an autobuilding Docker image.
 
-Regardless of how you made your tool, this tutorial will assume you are using CWL. For all other languages, :doc:`please see our documentation on workflows instead </getting-started/dockstore-workflows>`.
-
-.. note:: The WDL community does not explicitly differentiate :doc:`tools versus workflows </getting-started/intro-to-dockstore-tools-and-workflows>`. However, Dockstore allows the registration of "WDL tools" using legacy registration methods. We encourage people to :doc:`register WDLs as workflows </getting-started/dockstore-workflows>` instead.
+.. note:: Dockstore allows the registration of "WDL tools" using legacy registration methods. The WDL community does not explicitly differentiate :doc:`tools versus workflows </getting-started/intro-to-dockstore-tools-and-workflows>`. We encourage people to :doc:`register WDLs as workflows </getting-started/dockstore-workflows>` instead.
 
 Register Your Tool in Dockstore
 -------------------------------
 .. include:: /getting-started/github-apps/note--registration.rst
 
-There are a variety of ways to get your tools into Dockstore. Users can either use GitHub App registration or our legacy registration methods.
-GitHub App registration is the recommended way to register all new tools on Dockstore. GitHub App tools and tools registered using our other methods (legacy tools) are
-very different from one another. Use the following questions to determine which method to use:
+We recommend users register their tool with our GitHub App to benefit from GitHub's features and automatic syncing. If you do not wish to use GitHub, we do support other methods of registration in our legacy registration methods. Use the following questions to determine which registration method works for you:
 
 .. include:: /getting-started/how-to-register-work.rst
 
-If you must use the legacy tool registration methods, then you may want to read :doc:`Dockstore Tools Overhaul </../advanced-topics/dockstore-tools-overhaul>` before continuing
```

Comment:
I agree with removing this part out of this section, but I think there needs to be a link to Dockstore Tools Overhaul somewhere in the section directly under the legacy tools registration header on line 74.

Reply:
I disagree- If getting started is aimed at new users, why do they need to read about what we previously did? I don't think returning users will be heavy users of this page.

Reply:
The problem is that legacy registration creates legacy tools, which are very different from our new definition of a tool. So anyone using legacy registration needs to know the history.

Reply:
Ok, thanks for clarifying. I added a link to line 80. I think adding there helps keep it from disrupting the main tutorial flow.